### PR TITLE
Apply the static CORS only for static paths

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cgeoportal/tests/test_cachebuster.py [hunk 1]
```diff
@@ -57,5 +60,6 @@ def test_noreplace(self):
         request = MyRequest("/test2/123456/build.css")
         response = ctf(request)
         self.assertEqual(request.path_info, "/test2/123456/build.css")
-        self.assertEqual(response.headers["Access-Control-Allow-Origin"], "*")
-        self.assertEqual(response.headers["Access-Control-Allow-Headers"], "X-Requested-With, Content-Type")
+        # test no CORS
+        self.assertFalse("Access-Control-Allow-Origin" in response.headers)
+        self.assertFalse("Access-Control-Allow-Headers" in response.headers)
```
Comment (pvalsecc) [inline c2cgeoportal/tests/test_cachebuster.py]: self.assertNotIn

File: c2cgeoportal/lib/cacheversion.py [hunk 2]
```diff
@@ -61,7 +61,8 @@ def __call__(self, request):
 
         response = self.handler(request)
 
-        response.headers["Access-Control-Allow-Origin"] = "*"
-        response.headers["Access-Control-Allow-Headers"] = "X-Requested-With, Content-Type"
+        if path[1] in CACHE_PATH:
```
Comment (pvalsecc) [inline c2cgeoportal/lib/cacheversion.py]: I'll need more explanations, here...

File: c2cgeoportal/tests/test_cachebuster.py (hunk 1 above)
Comment (sbrunner) [inline c2cgeoportal/tests/test_cachebuster.py]: Tanks :-)

File: c2cgeoportal/lib/cacheversion.py (hunk 2 above)
Comment (sbrunner) [inline c2cgeoportal/lib/cacheversion.py]: Here we fill this global value with the static view name (second path element) https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/__init__.py#L343-L351
Already used 7 lines above :-)

File: c2cgeoportal/tests/test_cachebuster.py (hunk 1 above)
Comment (sbrunner) [inline c2cgeoportal/tests/test_cachebuster.py]: done

File: c2cgeoportal/lib/cacheversion.py (hunk 2 above)
Comment (pvalsecc) [inline c2cgeoportal/lib/cacheversion.py]: Yes, but I don't understand why you do that only if path[1] in CACHE_PATH?

And, bonus question, why not in the previous if with exactly the same condition?

Comment (sbrunner) [inline c2cgeoportal/lib/cacheversion.py]: i.-e.:
the `request.path_info` can have the value: `"/proj-ngeo/5f5e11b65b774343945a44eea2882d5d/build/mobile.css"`
the path will have the value `['', 'proj-ngeo', ...]`
the `CACHE_PATH` will have the value `[u'static', u'project', u'proj', u'proj-ngeo']`
...
For the dynamic view We can ash for user data and credentials, than we needs something dynamic that done here: https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/lib/caching.py#L109-L155

Bonus: to have the response :-)

**Comment by pvalsecc** [comment]
OK, ship it

**Pull request merged**